bin/network-monitor/README.md

```diff
@@ -137,6 +178,48 @@ The web dashboard provides a clean, responsive interface with the following feat
 - **Interactive Elements**: Copy-to-clipboard functionality for genesis commitments, transaction IDs, and note IDs
 - **Responsive Design**: Optimized for both desktop and mobile viewing
 
+## Account Management
+
+The monitor automatically manages Miden accounts for testing and monitoring purposes:
+
+### Created Accounts
+
+**Wallet Account:**
+- Uses RpoFalcon512 authentication scheme
+- Contains authentication keys for transaction signing
+- Automatically created if not present
+
+**Counter Program Account:**
+- Implements a simple counter with increment functionality
+- Includes authentication logic that restricts access to the wallet account
+- Uses custom MASM script with account ID-based authorization
+- Automatically created if not present
+
+### Account File Management
+
+The monitor automatically:
+1. Checks for existing account files on startup
+2. Creates new accounts if files don't exist
+3. Deploys accounts to the network via RPC
+4. Uses the specified file paths (default: `wallet_account.bin` and `counter_program.bin`)
```

Comment:
nit: When using `AccountFile`, we've used the `.mac` suffix for files. Also, I'd mention that the files are saved in the paths:
```suggestion
4. Saves the wallet and counter contract account in the specified file paths (default: `wallet_account.mac` and `counter_program.mac`)
```